crates/aide/src/axum/outputs.rs

```diff
@@ -122,3 +127,31 @@ impl<T> OperationOutput for Html<T> {
         }
     }
 }
+
+impl OperationOutput for JsonRejection {
+    type Inner = Self;
+
+    fn operation_response(ctx: &mut GenContext, operation: &mut Operation) -> Option<Response> {
+        Json::<String>::operation_response(ctx, operation)
```

Comment:
This means that the response is a JSON string, e.g. `"error happened"` with `application/json` content type, is that correct?

Reply:
Thank you!!

I fixed.